configs/xapsd/xapsd.yaml

```diff
@@ -17,6 +18,7 @@ port: 11619
 tlsCertfile:
 tlsKeyfile:
 tlsPort: 11620
```

Comment:
Just a very minor thing, but from a user perspective it would be nice to swap port and addr here, to have them in the order they are *usually* read in.

Reply:
Order changed as suggested!